tests/test_frame.py

```diff
@@ -2521,3 +2521,9 @@ def test_align() -> None:
     aligned_df0, aligned_df1 = df0.align(df1)
     check(assert_type(aligned_df0, pd.DataFrame), pd.DataFrame)
     check(assert_type(aligned_df1, pd.DataFrame), pd.DataFrame)
+
+
+def test_loc_new() -> None:
```

Comment:
Can you name this as `test_loc_returns_series()`

Reply:
done sir